index.html

```diff
@@ -989,49 +980,43 @@
           <ol>
             <li>Let <em>P</em> be a new <a>Promise</a>.
             </li>
-            <li>Return <em>P</em>.
+            <li>Return <em>P</em>, but continue running these steps in
+            parallel.
```

Comment:
Similar comment as above; there is likely little benefit to running the following steps in parallel, as the algorithm will terminate once the first matching PresentationConnection is found.  I think just dropping "in parallel" will address any confusion.